lib/active_model_serializers/adapter/json_api/deserialization.rb

```diff
@@ -93,8 +93,11 @@ def parse(document, options = {})
           attributes['id'] = primary_data['id'] if primary_data['id']
           relationships = primary_data['relationships'] || {}
 
-          filter_fields(attributes, options)
-          filter_fields(relationships, options)
+          attributes = transform_keys(attributes, options)
+          relationships = transform_keys(relationships, options)
```

Comment:
though, that would look wierd

            parsed_hash = ActiveModelSerializers::Deserialization.jsonapi_parse(
              params,
              only: [:image_height, :image_width],
              key_transform: :camel_lower
            )
            render json: parsed_hash